heracles/dices/cls.py

```diff
@@ -0,0 +1,95 @@
+# DICES: Euclid code for harmonic-space statistics on the sphere
+#
+# Copyright (C) 2023-2024 Euclid Science Ground Segment
+#
+# This file is part of DICES.
+#
+# DICES is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# DICES is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with DICES. If not, see <https://www.gnu.org/licenses/>.
+import numpy as np
+from copy import deepcopy
+from itertools import combinations
+from .bias_correction import correct_bias
+from .mask_correction import correct_mask
+from ..result import Result
+from ..mapping import transform
+from ..twopoint import angular_power_spectra
+
+
+def jackknife_cls(data_maps, vis_maps, jk_maps, fields, nd=1):
+    """
+    Compute the Cls of removing 1 Jackknife.
+    inputs:
+        data_maps (dict): Dictionary of data maps
+        vis_maps (dict): Dictionary of visibility maps
+        jkmaps (dict): Dictionary of mask maps
+        fields (dict): Dictionary of fields
+        nd (int): Number of Jackknife regions
+    returns:
+        cls (dict): Dictionary of data Cls
+    """
+    if nd < 0 or nd > 2:
+        raise ValueError("number of deletions must be 0, 1, or 2")
+    cls = {}
```

Comment:
Does this need to be a dictionary, or can it be a list?

Reply:
Yes! the delete2 correction uses the keys to make the computation easier to loop.